doc/developer/release-checklist.md

```diff
@@ -36,9 +41,11 @@
   ```shell
   tag=v<VERSION>-rc<N>
   git tag -a $tag -m $tag
-  git push origin $tag
+  git push upstream $tag
```

Comment:
I think we should just document that `origin` should be the `MaterializeInc/materialize` remote, I think our default instructions leave `origin` as that.

I think I'd change this to something like:

```
git push origin $tag  # where 'origin' is your MaterializeInc/materialize remote
```

Reply:
Okay! Will fix.

Personally, I've had `upstream` as my MaterializeInc/materialize remote and `origin` as my fork. I can't seem to find where we tell people that `origin` should point to Materialize, do you know where it is?

Reply:
> Personally, I've had upstream as my MaterializeInc/materialize remote
that might have happened during the great publification? 

When you `git clone URL` a repo that url is default marked as the `origin` remote, and then `git remote add NAME URL` marks that one as `NAME`, so depending on the order of clones you might have needed to come up with another name. `upstream` also sounds like something that I have called the same kind of remote that you're calling it, so I might have advised you to do this.

> Where we tell people that origin should point to Materialize

It's not explicit, but this clones `MaterializeInc/materialize`, although we should probably add another step that gets people to fork it and add the remote: https://github.com/MaterializeInc/mtrlz-setup/blob/794e513017/mac/materialize.sh#L88

Reply:
Ah got it, thank you!